app/src/main/kotlin/at/bitfire/davdroid/Constants.kt

```diff
@@ -34,6 +34,8 @@ object Constants {
 
     const val MANUAL_PATH_WEBDAV_MOUNTS = "webdav_mounts.html"
 
+    const val MANUAL_PATH_FIRST_STEPS = "introduction.html#first-steps"
```

Comment:
Looks good, but it shows a 404 because # is replaced by `%23`. I think the PATH and the section have to be split into two constants.